lib/vue/src/components/input/CInputPresort.vue

```diff
@@ -30,22 +30,13 @@
             class="align-middle"
             style="min-width: 250px;"
           >
-            <b-form-select
+            <c-input-select
```

Comment:
You probably need to import it in this component no?
It currently works because of global import on the webapp directly